src/test/silenceErrors.ts

```diff
@@ -0,0 +1,10 @@
+// silence container query errors
+const originalConsoleError = Object.assign({}, console.error);
+
+beforeAll(() => {
+  console.error = () => {};
```

Comment:
What's the reason for this guy? Are there a bunch of errors filling up the output?

Reply:
Get this annoying error over and over because it doesn't recognise container queries yet 😢 

![The terminal showing a long-winded error that it can't parse the CSS in one of the test iterations.](https://github.com/grafana/synthetic-monitoring-app/assets/6906380/a98e131d-dfa4-415e-b2d6-3d08846c9328)

![A zoomed out view of the terminal with a long list of errors creating noise for each test containing CSS container queries.](https://github.com/grafana/synthetic-monitoring-app/assets/6906380/e790b7be-6613-4bd9-880a-115d7ddb7bfa)

Reply:
woof that's pain